README.md

```diff
@@ -60,3 +60,9 @@ why-partner:
 Finally, side navigation menus are logically associated with primary menu items
 by matching the `href` of the primary menu item to the identifier of the side
 navigation menu; e.g., `/why-partner/` and `why-partner`.
+
+## Adding Javascript
+
+Store any additional code in the `scripts` directory - `webpack` bundles everything
+there into `assets/js/bundle.js`, which `jekyll` automatically builds to
+`_site/assets/js/bundle.js`.
```

Comment:
So my expectation is that directories like `scripts/` are for utilities that we use to develop the site, maybe something like `assets/src` or `src` would be a clearer way to say "this file is supposed to be bundled & deployed"

Reply:
Changing to src/js

Reply:
ok! The comment is out of date still and references `scripts` still